settings.gradle

```diff
@@ -64,9 +64,22 @@ rootProject.name = 'airbyte'
 // definition for dependency resolution
 dependencyResolutionManagement {
     repositories {
-        maven {
-            url 'https://airbyte.mycloudrepo.io/public/repositories/airbyte-public-jars/'
-        }
+        // # Gradle looks for dependency artifacts in repositories listed in 'repositories' blocks in descending order.
+
+        // Start with the local filesystem.
+        mavenLocal()
```

Comment:
Love that this is by default. I imagine it would be a welcome addition to the platform code base.